src/lib/homestore_backend/heap_chunk_selector.cpp

```diff
@@ -22,7 +22,7 @@ void HeapChunkSelector::add_chunk(csharedChunk& chunk) { m_chunks.emplace(VChunk
 void HeapChunkSelector::add_chunk_internal(const chunk_num_t chunkID) {
     if (m_chunks.find(chunkID) == m_chunks.end()) {
         // sanity check
-        LOGWARN("No chunk found for ChunkID {}", chunkID);
+        LOGWARNMOD(homeobject, "No chunk found for ChunkID {}", chunkID);
```

Comment:
HeapChunkSelector currently does not include homeobject headers...so this is left alone,